release-team/README.md

```diff
@@ -185,3 +185,19 @@ is undefined.
 ### During "Security" releases
 For all unplanned or embargoed releases
 - Facilitate security releases following the under the [Security Release Process](security-release-process-documentation/security-release-process.md)
+
+---
+
+## Milestone Maintainers
+
+Across release cycles, one of the best signals for [issue triage](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#milestones) is whether or not an issue or PR is actually targeted for the current milestone.
+
+To maintain up-to-date status on milestone inclusion, we rely on a set of Milestone Maintainers (members of the `milestone-maintainers` GitHub team) to apply the appropriate labels to issues / PRs. This is facilitated by `/milestone` commands and bot automation.
+
+Each release cycle, the current Release Team should seek to manage membership to the aforementioned GitHub team.
+
+The `milestone-maintainers` GitHub team should contain:
+- At least 2 SIG leads (either SIG Chairs or SIG Technical Leads) with Maintainer access to manage membership to the GitHub team (for their SIG)
+- A set of SIG Milestone delegates (known SIG feature reviewers / approvers), at least 2, which will work to keep issues / PRs / [feature tracking issues](https://github.com/kubernetes/features/issues) up-to-date for the current milestone
```

Comment:
Is the above two per SIG, or two in the whole project?  If the former, that's 2 leads and 2 delegates, or four per SIG, or 124 people total.

If it'sffour for the whole project, that doesn't seem nearly enough.

Suggest instead: at least one sig lead

Reply:
@jberkus how about at least one lead and at least two delegates?

Reply:
I think this should have a parallel commit on the sig charter template, existing sig charters, and get added to the currently in flight sig charters. Ie: go through top level and affirm SIGs know what they’re being signing up for and have it documented in their operations. Otherwise the ongoing management is less likely to happen.

With respect to numbers I’m ambivalent. I can see saying at least one SIG lead. And noting the operation choice is up to each SIG. They could have their lead(s) do all /milestone management, or add/remove additional delegates to spread their load. Pick the preferred one for the SIG, document how the SIG operates, get on with things.

For what I imagine the volume to be, it’s reasonable to say “SIG leads contact the release lead with adds/removes” instead of two sig folks with list maintainer privs at all times. Again the key is to have clear expectation and communication, versus people out of the blue requesting access without supporting justification from a sig. 

So I’d go simpler, leave maintenance mechanics on GH group to release lead, but clarify SIGs are responsible for picking a policy and telling the release lead what they want for changes. This way there’s four changes to maintainers of the list per year (four releases) instead of a lot more probably, and more clear ownership of 1 versus 124 owners.

Reply:
As in SIG Leads, as well as the RL, can add new members?  Seems fine, but then we need to give them some guidance.  I suggest "maximum of 3 members per SIG".